tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -30,7 +33,7 @@
 from gt4py.next.program_processors.runners import gtfn
 from gt4py.next.type_system import type_specifications as ts
 from gt4py.next import utils as gt_utils
-
+from gt4py.next import config
```

Comment:
Why are these imports added, when the code has not been changed? If they were necessary, the tests would be failing in main, wouldn't they?